connect.js

```diff
@@ -97,7 +96,7 @@ const wrap = (Wrapped, module, epics, logger, options = {}) => {
       });
     }
 
-    componentWillReceiveProps(nextProps) {
+    componentWillReceiveProps(nextProps) { // eslint-disable-line react/no-deprecated
```

Comment:
I wonder if we should instead use `UNSAFE_componentWillReceiveProps` here. I worry we may forget about this one if we disable linting. What do you think?

Reply:
Definitely don't mask the problem by using `UNSAFE_*`. We should either fix it, or leave the niggle in place so we keep feeling the discomfort until we do. Otherwise, we'll find things stop working all at once when we upgrade to React 17.

Reply:
I'll see if we can get https://issues.folio.org/projects/STCON/issues/STCON-70 onto somebody's plate soon.